index.rst

```diff
@@ -1,32 +1,24 @@
-:tocdepth: 1
+########################################################################################
+Investigation into vibrations in the M1M3 surrogate on the TMA due to the Fan Coil Units
+########################################################################################
 
-.. sectnum::
+.. abstract::
 
-.. Metadata such as the title, authors, and description are set in metadata.yaml
+   This technote uses data from the VMS, dc accelerometers and IMS to estimate the effect of the Fan Coil Units (FCUs) are likely to have on image quality
```

Comment:
Could you replace `VMS` with `Vibration Monitoring System (VMS)`?

Reply:
Same with `IMS` and `Independent Measurement System (IMS)`?

Reply:
done